src/components/Column.js

```diff
@@ -0,0 +1,94 @@
+const React = require('react');
+
+const Column = React.createClass({
+  propTypes: {
+    breakLg: React.PropTypes.number,
+    breakMd: React.PropTypes.number,
+    breakSm: React.PropTypes.number,
+    colLg: React.PropTypes.number,
+    colMd: React.PropTypes.number,
+    colSm: React.PropTypes.number,
+    columnCount: React.PropTypes.number,
+    offLg: React.PropTypes.number,
+    offMd: React.PropTypes.number,
+    offSm: React.PropTypes.number,
+    ordLg: React.PropTypes.number,
+    ordMd: React.PropTypes.number,
+    ordSm: React.PropTypes.number,
```

Comment:
Was there a reason for abbreviating all the props?  As some one new to this component, this might be hard to follow.